modules/css-converter/routes/variables-route.php

```diff
@@ -0,0 +1,173 @@
+<?php
+namespace Elementor\Modules\CssConverter\Routes;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use Elementor\Modules\CssConverter\Parsers\CssParser;
+use Elementor\Modules\CssConverter\Exceptions\CssParseException;
+use Elementor\Modules\CssConverter\Variable_Conversion_Service;
+use WP_REST_Request;
+use WP_REST_Response;
+
+class VariablesRoute {
```

Comment:
Probably it is better to use the existing Elementor approach to create custom endpoints.